ecosystem/sep-0006.md

```diff
@@ -331,6 +331,7 @@ Name | Type | Description
 `amount` | string | (optional) The amount of the asset the user would like to deposit with the anchor. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
 `country_code` | string | (optional) The [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) code of the user's current address. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
 `claimable_balance_supported` | string | (optional) `true` if the client supports receiving deposit transactions as a claimable balance, `false` otherwise.
+| `customer_id`                 | string | (optional) ID of an account associated with this user (identified by JWT) used by the anchor. If the anchor supports [SEP-12], the `customer_id` field should match [SEP-12] Customer's id.                                                                                                                                                                                                                                                                                               |
```

Comment:
I don't quite understand the definition.

> ID of an account associated with this user (identified by JWT) used by the anchor.

Is the `account` supposed to be some kind of off-chain record, like a github account? Or are you referring to a Stellar account here?

When you say `(identified by JWT)`, are you referring to the user or their account mentioned above? 

Unrelatedly, lets make sure we're being consistent with how we use "id" vs. "ID` and remove the capital C in `Customer's`.

Reply:
> Is the account supposed to be some kind of off-chain record, like a github account

Yes, off-chain account (internal anchor's account)

> When you say (identified by JWT), are you referring to the user or their account mentioned above?

Account + memo combination 

I will rephrase description so it's more clear